portfolio/src/main/webapp/index.html

```diff
@@ -70,13 +70,23 @@ <h2 style= "margin: 0 5% 0 5%">Princeton University</h2>
 
 
                 <!-- ***************************************SERVLETS WEEK-3*************************************** -->
-
+                
                     <h1>Fetch</h1>
                     <p>Click here to see Favourite Ice-creams:</p>
                     <button onclick="getJson()">Fetch statement!</button>
                     <div id="gen-container"></div>
 
                 <!-- Form -->
+                <form action="/reverse" method="POST">
+
+                    <p>Enter a word to see its reverse:</p>
+                    <textarea name="text-input">teem</textarea>
```

Comment:
Is this a typo? You don't need to have any text in the <textarea>

Reply:
It was an example of the types of words people could perhaps try entering. 'teem' would be flipped to 'meet'

Reply:
Ooh ok makes sense